code/style.css

```
@@ -1,4 +1,260 @@
-html {
-  background: #0025ff;
-  color: #fff;
+* {
+  box-sizing: border-box;
 }
+
+body { 
+  background: rgb(42,44,49);
+  background: linear-gradient(90deg, rgba(42,44,49,1) 0%, rgba(35,40,54,1) 34%, rgba(52,63,94,1) 100%);
+  color: whitesmoke;
+  font-family: Open Sans;
+  margin: 0 auto;
+}
+
+h1 {
+  font-family: Lato, serif;
+  font-weight:100;
+  font-size: 58px;
+  letter-spacing: 0.05em;
+  margin: 0;
+  display:inline;
+}
+
+h2{
+  font-family: Lato, sans-serif;
+  font-size: 30px; 
+  margin:20px, 20px;
+}
+
+h3 {
+  font-family: Lato, sans-serif;
+  font-size: 20px;
+  margin-top: 10px;
+  margin-bottom: 0;
+}
+
+h4 {
+  font-family: Lato, sans-serif;
```

When you are using fonts that are not installed by default in your computer (for example Arial is it) the best practice is to add `'` around the font name
```suggestion
  font-family: 'Lato', sans-serif;
```